.github/workflows/lint-test.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         if: steps.list-changed.outputs.changed == 'true'
         run: |
           helm env
-          helm plugin install https://github.com/quintush/helm-unittest.git --version 0.2.8
+          helm plugin install https://github.com/quintush/helm-unittest.git --version 0.2.11
```

Comment:
The plugin failed to install in CI, probably due to the to some intermittent issue related to github. Let me know if i should revert it. I think it doesn't do harm bumping this one, too. 

There weren't any breaking changes, just some smaller improvements, see releases: https://github.com/quintush/helm-unittest/releases